packages/paste-core/components/combobox/stories/index.stories.tsx

```diff
@@ -508,21 +508,30 @@ ComboboxOverflowLongValue.story = {
 };
 
 export const ComboboxControlled = (): React.ReactNode => {
-  const [value, setValue] = React.useState('');
-  const [selectedItem, setSelectedItem] = React.useState({});
+  const [value, setValue] = React.useState('United Arab Emirates');
+  const [selectedItem, setSelectedItem] = React.useState({code: 'AE', label: 'United Arab Emirates', phone: '971'});
   const [inputItems, setInputItems] = React.useState(objectItems);
   return (
     <>
+      <Box paddingBottom="space70">
+        Input value state: {JSON.stringify(value)}
+        <br />
+        Selected item state: {JSON.stringify(selectedItem)}
+      </Box>
+
       <Combobox
         autocomplete
         items={inputItems}
         labelText="Choose a country:"
         helpText="This is the help text"
-        optionTemplate={(item: ObjectItem) => (
-          <div>
-            {item.code} | {item.label} | {item.phone}
-          </div>
-        )}
+        optionTemplate={(item: ObjectItem) => {
```

Comment:
I noticed something in these stories when clicking around, and then saw it in subsequent ones, just wanted to check if it was expected behavior. Loom here: https://www.loom.com/share/52cfc26cc5154f8b98a6254b48f0921a

Reply:
Yeah it can be a little funky, it is intentional but not clear.

This is just the way we've implemented our stories. You get to choose how to display the list option. So in autocomplete comboboxes we filter the list of options based on what you type or what you select. The persist it on reopen. 

No _wrong_, not _right_. Just "a way" to do it. Good spot though